mobsf/templates/general/recent.html

```diff
@@ -77,14 +77,14 @@ <h3 class="box-title"><i class="fa fa-rocket"></i> Recent Scans</h3>
                             <td>{{ e.MD5 }}</td>
                             <td>{{ e.TIMESTAMP }}</td>
                             <td><p>
-                                    <a class="btn btn-primary btn-sm" style="margin-bottom: 3px;" href="../{{ e.ANALYZER }}/?name={{e.FILE_NAME}}&amp;checksum={{e.MD5}}&amp;type={{e.SCAN_TYPE}}"><i class="fas fa-eye" title="View static analysis details"></i> Static Details</a><br/>
+                                    <a class="btn btn-primary btn-sm" style="margin-bottom: 3px; width: 125px;" href="../{{ e.ANALYZER }}/?name={{e.FILE_NAME}}&amp;checksum={{e.MD5}}&amp;type={{e.SCAN_TYPE}}"><i class="fas fa-eye" title="View static analysis details"></i> Static Details</a><br/>
                                     {% if e.PACKAGE_NAME %}
-                                    <a class="btn btn-primary btn-warning btn-sm" style="margin-bottom: 3px;" href="../appsec_dashboard/{{ e.MD5 }}/"><i class="fas fa-user-shield" title="View security scorecard"></i> Scorecard</a><br/>
-                                    <a class="btn btn-primary btn-danger btn-sm" href="{{dependency_track_url}}/projects/?tag={{ e.MD5 }}"><i class="fas fa-link" title="View third party dependency details"></i> Dependencies</a><br/>
+                                    <a class="btn btn-primary btn-warning btn-sm" style="margin-bottom: 3px;  width: 125px;" href="../appsec_dashboard/{{ e.MD5 }}/"><i class="fas fa-user-shield" title="View security scorecard"></i> Scorecard</a><br/>
+                                    <a class="btn btn-primary btn-danger btn-sm" style="width: 125px;" href="{{dependency_track_url}}/projects/?tag={{ e.MD5 }}"><i class="fas fa-link" title="View third party dependency details"></i> Dependencies</a><br/>
                                     {% endif %}
                                 </p><p>
                                    <a class="btn btn-outline-primary btn-sm" href="../pdf/?md5={{ e.MD5 }}"><i class="fas fa-file-pdf" title="Generate PDF report of static analysis details"></i></a>
-                                   <a class="btn btn-outline-info btn-sm" href="../pdf-sbom/?md5={{ e.MD5 }}"><i class="fas fa-file-pdf" title="Generate PDF report of third party dependency details"></i></a>
+                                   <!--<a class="btn btn-outline-info btn-sm" href="../pdf-sbom/?md5={{ e.MD5 }}"><i class="fas fa-file-pdf" title="Generate PDF report of third party dependency details"></i></a>-->
```

Comment:
Are we going to add this back in when there is an SBOM to present?

Reply:
The relevant link is just commented out for now and can be uncommented once we get the sboms working. We may need to look into figuring out if there is one present in the future.